pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -52,6 +53,12 @@ func WithListUsersQueryLogger(l logger.Logger) ListUsersQueryOption {
 	}
 }
 
+func WithListUserMaxResults(max uint32) ListUsersQueryOption {
+	return func(rq *listUsersQuery) {
+		rq.maxResults = max
+	}
+}
```

Comment:
**Question:** Will the rest of the max results be implemented in a future PR?

Reply:
Yes, I will start that today